doc/Command-Reference.md

```diff
@@ -4088,6 +4092,170 @@ This command deletes the SNMP Trap server IP address to which SNMP agent is expe
+	  "Ethernet0": {
+	      "STATUS": "auto",
+	      "HEALTH": "HEALTHY"
+	  }
+      }
+  }
+  ```
+
+**show muxcable config**
+
+This command displays all the configurations of either all the ports which are connected to muxcable or any individual port selected by the user. The resultant table or json output will show the current configurations of muxcable on the port(active/standby) and also the ipv4 and ipv6 address of the port as well as peer TOR ip address with the hostname.
+
+- Usage:
+  ```
+  show muxcable config [OPTIONS] [PORT]
+  ```
+With no optional argument, all the ports muxcable configuration will be displayed in tabular form
+While displaying the muxcable configuration, users can configure the following fields
+PORT   optional - Port name should be a valid port
+--json optional -  option to display the result in json format. By default output will be in tabular format.
+
+- Example:
+  ```
+  admin@sonic:~$ show muxcable config
+  SWITCH_NAME    PEER_TOR
+  -------------  ----------
+  sonic          10.1.1.1
+  port       state    ipv4      ipv6
+  ---------  -------  --------  --------
+  Ethernet0  active  10.1.1.1  fc00::75
+
+  admin@sonic:~$ show muxcable config --json
+  {
+      "MUX_CABLE": {
+	  "PEER_TOR": "10.1.1.1",
+	  "PORTS": {
+	      "Ethernet0": {
+		  "STATE": "active",
+		  "SERVER": {
+		      "IPv4": "10.1.1.1",
+		      "IPv6": "fc00::75"
+		  }
+	      }
+	  }
+      }
+  }
+
+  admin@sonic:~$ show muxcable config Ethernet0
+  SWITCH_NAME    PEER_TOR
+  -------------  ----------
+  sonic          10.1.1.1
+  port       state    ipv4      ipv6
+  ---------  -------  --------  --------
+  Ethernet0  active  10.1.1.1  fc00::75
+
+  admin@sonic:~$ show muxcable config Ethernet0 --json
+  {
+      "MUX_CABLE": {
+	  "PORTS": {
+	      "Ethernet0": {
+		  "STATE": "active",
+		  "SERVER": {
+		      "IPv4": "10.1.1.1",
+		      "IPv6": "fc00::75"
+		  }
+	      }
+	  }
+      }
+  }
+  ```
+
+
+### Muxcable Config commands
+
+
+**config muxcable mode**
+
+This command is used for setting the configuration of a muxcable Port/all ports to be active or auto. The user has to enter a port number or else all to make the muxcable config operation on all the ports. Depending on the status of the muxcable port state the resultant output could be OK or INPROGRESS . OK would imply no change on the state, INPROGRESS would mean the toggle is happening in the background.
+
+- Usage:
+  ```
+  config muxcable mode [OPTIONS] <operation_status> <port_name>
+  ```
+
+While configuring the muxcable, users needs to configure the following fields for the operation
+<auto/active>  - operation_state, permitted operation to be configured which can only be auto or active
+PORT           - Port name could be a valid port or all to which the muxcable operation would occur. All would imply the operation on all the ports
+--json         - option to display the result in json format. By default output will be in tabular format.
```

Comment:
Check the rendered Markdown. There are no line breaks before these lines. Maybe we should make them bullet points?

Reply:
converted them bullet points now